src/hypothesis/internal/conjecture/data.py

```diff
@@ -111,7 +112,7 @@ def __init__(self, max_length, draw_bytes):
         self.example_stack = []
         self.has_discards = False
 
-        self.start_example()
+        self.start_example(calc_label_from_name('top'))
```

Comment:
I think these `calc_label_from_name` calls should probably go in top level constants. They're somewhat expensive to calculate. Elsewhere we cache them on the class object, but that's not available here. You could also add a local method that caches them on a dict in the module if you prefer (it's certainly more readable that way)